defender/staged/downloaded/Forta_Governance.spec.js

```diff
@@ -196,9 +227,16 @@ describe('check autotask', () => {
     );
     // run the autotask on the events
     await handler(autotaskEvent);
+
+    const data = '{"content":"**Executed Proposal**  ✅"}';
```

Comment:
Is this really a representative Discord post?

It looks like you might need to mock the `axios.get` call so that you can mock the response from the Compound API. That would allow you to return a mocked proposal name that _should_ show up in the Discord post.